spec/cypress/integration/admin/AdminManagement.feature

```diff
@@ -3,7 +3,7 @@ Feature: Admin user modifying admin users
 
   Background:
     Given I am logged in as an "admin"
-    And scenario "admin/administrators/manage_admin_users" has been run
+    And Admin account was created with email "emma-dow@example.com" and name "Emma Dow"
```

Comment:
We've been using scenario files for this - I think it would be better to continue that pattern